src/main/java/core/basesyntax/service/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvFileWriterServiceImpl implements FileWriterService {
+    private static final String HEADERS = "fruit,quantity";
+
+    @Override
+    public void writeToFile(String fileName, String report) {
+        File file = new File(fileName);
```

Comment:
If I am not mistaken, we need to create file automatically, not by hand
file.createNewFile()

Reply:
it's already create a file automatically, thanks))